### ISSUE TITLE
Browser Notification Support

### ISSUE THREAD

**Issue by md2100** [regolith-linux/regolith-desktop#211]
First of all, This distro rocks! I am really digging 1.3, and the notification system is great!

1 thing I have noticed, is browser notifications don't come through the new system. Are there any plans to support that?  I'm limited to using Outlook Online (O365) for work, and much prefer this notification system to something a little more aggressive (like Dunst), so having a working notification that is... more "sticky" would be rad!

I'd be willing to attempt to help if someone can give me a little direction as to what is needed!

**Comment by md2100** [comment]
I Think I found at the issue (at least in part): [https://bbs.archlinux.org/viewtopic.php?id=252375]

Rotification does not support "actions" which I believe is the reason firefox & chrome are choosing to use their own built in notification system instead.

**Comment by webbson** [comment]
Seems like it yeah. Was thinking about why Brave notifications came in a the browser instead when I switched to Regolith.

**Comment by kgilmer** [comment]
Interesting, I'll take a look.

**Comment by sr9992** [comment]
I experimented with this a couple days ago when I noticed that Firefox was falling back to it's built-in notifications rather than using Rofication.  I blindly copied the capabilities that [dunst](https://github.com/dunst-project/dunst/blob/master/src/dbus.c) reports to dbus, and Firefox is now using Rofication:

rofication-daemon.py:
```python
    def GetCapabilities(self):
        return [
            "actions",
            "body",
            "body-hyperlinks",
            "body-markup",
        ]
```

I'm not sure if there are any consequences to telling dbus that we support notification features that we truly don't, but it's a start.

Also, really been enjoying Regolith!  Keep up the great work!

**Comment by kgilmer** [comment]
I missed your fix before, thanks for reporting @sr9992 !  I will look into this shortly.

**Comment by moritzheiber** [comment]
So, according to https://github.com/regolith-linux/regolith-rofication/blob/master/rofication/_dbus.py#L49 we support `actions` and `body`. I don't believe we'll have any regressions by supporting `body-markup` or `body-hyperlinks`. I'll prepare a PR.

**Comment by moritzheiber** [comment]
Interestingly enough, for me the notifications of Firefox are showing up in Rofication .. :thinking: 

@sr9992 is this still reproducible?

**Comment by sr9992** [comment]
No, it works for me now.  [This page](https://www.bennish.net/web-notifications.html) is really useful for testing the notifications.  I see notifications in Rofication in both Firefox and Chrome from that page.  I also tested Outlook Web Access using Firefox, and it appears to work.

I suspect it was fixed [here](https://github.com/regolith-linux/regolith-rofication/commit/28fa7efe0ca366557153e11dfd4d6e19077f82c4).  From my perspective, we can probably close this issue as fixed.
